lib/paper_trail/version_concern.rb

```diff
@@ -210,9 +212,8 @@ def reify(options = {})
     # not have an `object_changes` text column.
     def changeset
       return nil unless self.class.column_names.include? 'object_changes'
-
-      _changes = self.class.object_changes_col_is_json? ? object_changes : PaperTrail.serializer.load(object_changes)
-      @changeset ||= HashWithIndifferentAccess.new(_changes).tap do |changes|
+      _changes = HashWithIndifferentAccess.new(object_changes_deserialized)
+      @changeset ||= _changes.tap do |changes|
         if PaperTrail.serialized_attributes?
           item_type.constantize.unserialize_attribute_changes_for_paper_trail!(changes)
         end
```

Comment:
Good suggestion.  I like it, though I prefer to extract a new method rather than have an anonymous block that long.